### PR TITLE
Creating a stub of the file for San Diego

### PR DESCRIPTION
**Pull request by saldaoud**
Initial draft of the file

### CODE REVIEW

**Comment by sisterneely** [comment]
Have you hiked Cowles Mountain recently?

File: places/San-Diego.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## This file is for San Diego CA.
+
+### Great Places To Eat
```
Comment (kk121417) [inline places/San-Diego.md:3]: @orageno | :+1: how about Il Fornaio

Comment (saldaoud) [inline places/San-Diego.md:3]: @kk121417  I like Il Fornaio!  I prefer Karl Strauss :)

Comment (saldaoud) [inline places/San-Diego.md:3]: @sisterneely I have not hiked the mountain.  Black Mountain was my last hike!

**Pull request merged**